examples/github_stats/test_app.py

```diff
@@ -0,0 +1,41 @@
+# This file is not part of the example. It is a test file to ensure the example
+# works as expected during the CI process.
+
+
+import os
+import unittest
+from unittest import mock
+
+from fastapi.testclient import TestClient
+
+from ... import function_service
+from ...test_client import ServerTest
+
+
+class TestAutoRetry(unittest.TestCase):
+    @mock.patch.dict(
+        os.environ,
+        {
+            "DISPATCH_ENDPOINT_URL": "http://function-service",
+            "DISPATCH_API_KEY": "0000000000000000",
+        },
+    )
+    def test_foo(self):
+        from . import app
+
+        server = ServerTest()
+        servicer = server.servicer
+        app.dispatch._client = server.client
+        app.get_repo_info._client = server.client
+        app.get_contributors._client = server.client
+        app.main._client = server.client
+
+        http_client = TestClient(app.app, base_url="http://dispatch-service")
+        app_client = function_service.client(http_client)
+
+        response = http_client.get("/")
+        self.assertEqual(response.status_code, 200)
+
+        server.execute(app_client)
```

Comment:
Given the example I think this should be ran multiple times to verify that multiple functions are indeed called. This is possible as of https://github.com/stealthrocket/dispatch-sdk-python/pull/76. For example: https://github.com/stealthrocket/dispatch-sdk-python/blob/main/examples/auto_retry/test_app.py#L44

Reply:
Calling multiple execute doesn't give me a new response somehow.

Reply:
OK feel free to merge, I'll look into it. Might have missed something in #76.